doc/source_tech_preview/index.rst

```diff
@@ -1,12 +1,9 @@
-Welcome to socok8s repo
+Welcome to SUSE OpenStack Cloud on Kubernetes
```

Comment:
->SUSE Containerized OpenStack. That's the product that T.R. wants to use.

Reply:
nice. better.